externals/kyuubi-flink-sql-engine/src/test/scala/org/apache/kyuubi/engine/flink/operation/FlinkOperationSuite.scala

```diff
@@ -665,6 +665,18 @@ class FlinkOperationSuite extends WithFlinkSQLEngine with HiveJDBCTestHelper {
     })
   }
 
+  test("execute statement - set/get catalog") {
+    withJdbcStatement()({ statement =>
+      statement.executeQuery("create catalog cat_a with ('type'='generic_in_memory')")
+      val catalog = statement.getConnection.getCatalog
+      assert(catalog == "default_catalog")
+      statement.getConnection.setCatalog("cat_a")
+      val changedCatalog = statement.getConnection.getCatalog
+      assert(changedCatalog == "cat_a")
+      assert(statement.execute("drop catalog cat_a"))
+    })
+  }
+
   test("execute statement - create/alter/drop database") {
     // TODO: validate table results after FLINK-25558 is resolved
```

Comment:
@link3280 I see you comment it's fixed by FLINK-24685, should we update it?

Reply:
Yes, the comment could be deleted now.